.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - v4
+      - main
```

Comment:
Lets remove this from main. I'm a little nervous publishing to main on every commit right now

Reply:
removed